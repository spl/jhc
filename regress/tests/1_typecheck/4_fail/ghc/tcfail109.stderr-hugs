ERROR "tcfail109.hs":16 - Cannot build superclass instance
*** Instance            : Collects Bool Stupid
*** Context supplied    : ()
*** Required superclass : Eq Stupid
