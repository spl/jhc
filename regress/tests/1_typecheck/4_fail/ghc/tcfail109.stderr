
tcfail109.hs:16:10:
    No instance for (Eq Stupid)
      arising from the superclasses of an instance declaration
    Possible fix: add an instance declaration for (Eq Stupid)
    In the instance declaration for `Collects Bool Stupid'
